packages/stateManager/package.json

```diff
@@ -1,19 +1,34 @@
 {
-  "name": "stateManager",
+  "name": "@massmarket/stateManager",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "vitest",
+    "test:chromium": "vitest --browser=chromium",
+    "lint": "eslint --ext .ts .",
+    "build": "tsc -b"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
     "@massmarket/client": "workspace:*",
-    "@massmarket/tsconfig": "workspace:*",
     "@massmarket/schema": "workspace:*",
-    "abstract-level": "^2.0.0",
-    "level": "^8.0.1"
+    "@massmarket/tsconfig": "workspace:*",
+    "@massmarket/utils": "workspace:*",
+    "events": "^3.3.0",
+    "level": "^8.0.1",
+    "viem": "^2.16.3",
+    "vitest": "^2.0.4"
```

Comment:
`vitest` should be in devDep. Please check the rest to make sure that they are in the correct place